tools/release_do_nothing.py

```diff
@@ -0,0 +1,758 @@
+    # TODO: automate using setuptools_scm.
+    release_strings = None
+    message = (
+        "Input the release tag you are creating today, including any release "
+        "candidate suffix.\n"
+        "https://semver.org/\n"
+        "https://scitools-iris.readthedocs.io/en/latest/developers_guide/release.html?highlight=candidate#release-candidate"
+    )
+    while release_strings is None:
+        input_tag = _get_input(message, "e.g. v1.2.3rc0")
+        try:
+            release_strings = ReleaseStrings(input_tag)
+        except ValueError as err:
+            _report_problem(str(err))
+    return release_strings
+
+
+def check_release_candidate(
+    release_type: ReleaseTypes, release_strings: ReleaseStrings
+) -> bool:
+    is_release_candidate = "rc" in release_strings.tag
+
+    message = "Checking tag for release candidate: "
+    if is_release_candidate:
+        message += "DETECTED\nThis IS a release candidate."
+    else:
+        message += "NOT DETECTED\nThis IS NOT a release candidate."
+    _break_print(message)
+
+    if release_type == ReleaseTypes.PATCH and is_release_candidate:
+        message = (
+            "Release candidates are not expected for PATCH releases. "
+            "Are you sure you want to continue?"
+        )
+        if _get_input(message, "y / N").casefold() != "y".casefold():
+            exit()
+    return is_release_candidate
+
+
+def check_first_in_series(
+    release_type: ReleaseTypes,
+    release_strings: ReleaseStrings,
+    is_release_candidate: bool,
+) -> bool:
+    first_in_series = False
+    if release_type != ReleaseTypes.PATCH:
+        message = (
+            "Have there been any prior releases in the "
+            f"{release_strings.series} series, including release candidates?"
+        )
+        first_in_series = (
+            _get_input(message, "Y / n").casefold() == "n".casefold()
+        )
+        if first_in_series:
+            _break_print("First in series confirmed.")
+            if not is_release_candidate:
+                message = (
+                    "The first release in a series is expected to be a "
+                    "release candidate, but this is not. Are you sure you "
+                    "want to continue?"
+                )
+                if _get_input(message, "y / N").casefold() != "y".casefold():
+                    exit()
+        else:
+            _break_print("Existing series confirmed.")
+    return first_in_series
+
+
+def update_standard_names(first_in_series: bool) -> None:
+    if first_in_series:
+        message = (
+            "Update the file ``etc/cf-standard-name-table.xml`` to the latest CF "
+            "standard names, via a new Pull Request.\n"
+            "(This is used during build to automatically generate the sourcefile "
+            "``lib/iris/std_names.py``).\n"
+            "Latest standard names: http://cfconventions.org/Data/cf-standard-names/current/src/cf-standard-name-table.xml"
```

Comment:
Perhaps adding the line of code `wget "http://cfconventions.org/Data/cf-standard-names/current/src/cf-standard-name-table.xml"` just clarifies how best to do this?

Reply:
> Perhaps adding the line of code `wget "http://cfconventions.org/Data/cf-standard-names/current/src/cf-standard-name-table.xml"` just clarifies how best to do this?

eb17a91a4

Reply:
> Also, mentioning adding a What's New might be helpful here, as I personally find this easy to forget.

It's OK to forget. There's a reminder in the Pull Request Checklist and I reckon that's enough. If we start adding reminders everywhere things are going to get quite noisy and have potential to be inconsistent.

https://github.com/SciTools/iris/blob/949b2968deceffcb1bcfc606b90e1ac97a1510bd/.github/pull_request_template.md?plain=1#L1-L9

Reply:
Yup, I'd agree in retrospect